src/HighlightCeption.php

```diff
@@ -85,102 +86,98 @@ public function _after(TestInterface $test)
     public function clickWithLeftButton($cssOfXPath = null, $offsetX = null, $offsetY = null)
     {
-        $this->highlightElement($cssOfXPath);
         $this->webDriverModule->clickWithLeftButton($cssOfXPath, $offsetX, $offsetY);
+        $this->highlightElement($cssOfXPath);
     }
 
     /**
      * @inheritdoc
      */
     public function clickWithRightButton($cssOfXPath = null, $offsetX = null, $offsetY = null)
     {
-        $this->highlightElement($cssOfXPath);
         $this->webDriverModule->clickWithRightButton($cssOfXPath, $offsetX, $offsetY);
+        $this->highlightElement($cssOfXPath);
     }
 
     /**
      * Highlight text on site
-     * 
+     *
      * @param string $text
      */
     private function highlightText($text)
     {
         try {
-            $this->loadJQuery();
             $this->debug('[Highlight Text] ' . $text);
-            $this->webDriverModule->executeJs('jQuery(document).ready(function (){
-                jQuery("body").highlight("' . $text . '");
-                ' . sprintf('jQuery(".highlight").css(%s);', $this->cssStyle) . '
-            });');
+            $el = $this->webDriver->findElement(WebDriverBy::xpath("//*[text()[contains(., '{$text}')]]"));
+            $this->webDriver->executeScript("let str = arguments[0].innerHTML.replace(/({$text})/g, '<span style=\"{$this->cssStyle}\">$1</span>'); arguments[0].innerHTML = str;", [$el]);
+            // $this->webDriver->executeScript("arguments[0].setAttribute('style', 'background-color: yellow; color: black;')", [$el]);
         } catch(Exception $e) {
             $this->debug(sprintf("[Highlight Exception] %s \n%s", $e->getMessage(), $e->getTraceAsString()));
         }
     }
 
     /**
      * Highlight element on site
-     * 
+     *
      * @param string|array $selector
      */
     private function highlightElement($selector)
     {
         try {
             $locator = $this->getSelector($selector);
             if ($locator) {
-                $this->loadJQuery();
                 if (Locator::isXPath($locator)) {
-                    $this->loadJQueryXPath();
-                    $this->debug('[Highlight XPath] ' . Locator::humanReadableString($locator));
-                    $this->webDriverModule->executeJs(sprintf('jQuery(document).xpath("%s").css(%s);', addslashes($locator), $this->cssStyle));
+                  $el = $this->webDriver->findElement(WebDriverBy::xpath($locator));
                 } else {
-                    $this->debug('[Highlight Selector] ' . Locator::humanReadableString($locator));
-                    $this->webDriverModule->executeJs(sprintf('jQuery("%s").css(%s);', addslashes($locator), $this->cssStyle));
-                }
+                  // assume css
+                  $el = $this->webDriver->findElement(WebDriverBy::cssSelector($locator));
+              }
+              $this->webDriver->executeScript("arguments[0].setAttribute('style', 'background-color: yellow; color: black;')", [$el]);
```

Comment:
see [comment](https://github.com/hyperia-sk/highlight-ception/pull/3/files/31c38558235614a60363a99533d7b92b0efe1f73#diff-172821954fd472dc8ebb88458bacf276R67)

Reply:
Forgot about this functionality. Added it back into the file. Thanks for pointing that out.